packages/react/rollup.config.mjs

```diff
@@ -127,7 +129,36 @@ const baseConfig = {
     }),
     importCSS({
       modulesRoot: 'src',
-      postcssPlugins: [],
+      postcssPlugins: [
+        customPropertiesFallback({
+          importFrom: [
+            () => {
+              const primitiveFallbacks = [
```

Comment:
I want to pull this in from a glob of the `node_modules/@primer/primitives/dist/fallbacks/` directory